RELEASE_NOTES.md

```diff
@@ -36,33 +38,33 @@ accompanying blog post.
 
 - New Sigmoid display transform module.
 
-- Two new highlight recovery algorithm are proposed: in-paint opposed
+- Two new highlight recovery algorithms are proposed: in-paint opposed
   and segmentation. The in-paint opposed has proved to be very stable
-  and good in a lot of pictures so it has been made the default now.
+  and good in a lot of pictures, so it has been made the default now.
 
 - A full rework of the image display has been done. An initial work
-  has been to create an easy to use image display on the central
-  area. This routine supports all display mode: focus-peaking and
+  has been to create an easy-to-use image display on the central
+  area. This routine supports both focus-peaking and
```

Comment:
I'm not sure about this change. Perhaps what was meant is that
it **supports, in all display modes, both focus-peaking and** color assessment.